tests/03-basic-ceos/01-two-ceos.robot

```diff
@@ -46,6 +46,13 @@ Ensure IPv6 default route is in the config file
     Log    ${f}
     Should Contain    ${f}    ipv6 route
 
+Ensure MGMT VRF is present
+    ${rc}    ${output} =    Run And Return Rc And Output
+    ... sudo docker exec -it clab-${lab-name}-${node1-name} /bin/Cli -p 15 -c "show vrf MGMT"
+    Log    ${output}
+    Should Be Equal As Integers    ${rc}    0
+    Should Contain  ${output}   MGMT
+
```

Comment:
I will fix this test @bewing

Reply:
@bewing 
I think the test functionally works now, but I am curious if the output is considered to be healthy?

```
time="2022-07-18T07:32:49Z" level=info msg="Parsing & checking topology file: 03-ceos01-clab.yml"
time="2022-07-18T07:32:50Z" level=info msg="Executed command 'Cli -p 15 -c 'show vrf MGMT'' on clab-03-01-two-ceos-n1. stdout:   VRF        RD              Protocols       State             Interfaces
---------- --------------- --------------- -------------------- -----------
    MGMT       <not set>       ipv4,ipv6       v4:incomplete,    Management0                                               v6:incomplete                                                              
```

i.e. I am not sure if `v4: incomplete` is expected?

Maybe you can try out a binary built with the latest commit and see it in action?

docker run --rm -v $(pwd):/workspace ghcr.io/oras-project/oras:v0.12.0 pull ghcr.io/srl-labs/clab-oci:b1bbf898

Reply:
The output from the command on 4.28.1F is
```
$ docker exec -it clab-mgmt-spine1 /bin/Cli -p 15 -c "show vrf MGMT"
   VRF        Protocols       State         Interfaces
---------- --------------- ---------------- ----------
   MGMT       IPv4            no routing    Ma0
   MGMT       IPv6            no routing    Ma0
```

I think `incomplete` just means "routing protocol not enabled".  Since we don't want/need to run a routing protocol in a management VRF by default, I believe this is OK.